process_report/processors/coldfront_fetch_processor.py

```diff
@@ -62,6 +64,13 @@ def _fetch_coldfront_allocation_api(self):
 
         return r.json()
 
+    def _get_coldfront_api_data(self):
+        if self.coldfront_data_filepath:
```

Comment:
maybe log an info message that we are reading the data from a file at the given path. This is just in case someone provides a file path AND the environment variables for coldfront_url/keycloak_stuff.